.csharpierrc

```diff
@@ -0,0 +1,6 @@
+{
+    "printWidth": 180,
```

Comment:
While opinions on the values of this [vary greatly](https://stackoverflow.com/questions/578059/studies-on-optimal-code-width) and I couldn't find a scientific answer, 180 seems higher than what everyone recommends. After much deliberation we at Lombiq settled with 120 characters like a decade ago, and it has been working well since, across generations of screen sizes (since it's rather dependent on human anatomy).